app/models/session_stats.rb

```diff
@@ -0,0 +1,32 @@
+class SessionStats
+  def initialize(patient_sessions:, location:)
+    @patient_sessions = patient_sessions
+    @location = location
+  end
+
+  def call
```

Comment:
Mixing in this service-style `call` into a model feels a little odd to me. I think it would fit the "model" pattern better if the stats were attributes.

Reply:
Agreed. Hopefully addressed in [17bcbc4](https://github.com/nhsuk/manage-childrens-vaccinations/pull/918/commits/17bcbc45ba395bbe8fd3b3fe18a5dbdbabc1a021)